Apigee/Mint/Developer.php

```diff
@@ -418,43 +418,25 @@ public function topUpPrepaidBalance($new_balance, $developer_or_company_id = nul
 
     public function getRevenueReport($report)
     {
-        $url = '/mint/organizations/'
-            . rawurlencode($this->config->orgName)
-            . '/developers/'
-            . rawurlencode($this->email)
-            . '/revenue-reports';
+        $url = rawurlencode($this->email) . '/revenue-reports';
```

Comment:
Did you mean to change the value of `$url` here? If so, an explanation would help.

Reply:
Yes, I wanted to change this value, since previously whole url was constructed here and used as temporary base url, but for `Apigee/Mint/Developer.php` class `$base_url` in constructor is defined as: 
```
public function __construct(OrgConfig $config)
    {
        $base_url = '/mint/organizations/' . rawurlencode($config->orgName) . '/developers';
```
and this means that we do not have to replace base url, but we can just append `rawurlencode($this->email) . '/revenue-reports';` to the base url value.
which will be equal to what was previously used but removes need to use methods and protects `$base_url` from exceptions
```
setBaseUrl()
restoreBaseUrl()
```

Reply:
thanks for the explanation, makes sense!